uk_election_timetables/elections/city_of_london_local.py

```diff
@@ -21,12 +89,44 @@ def __init__(self, poll_date: date):
     @property
     def sopn_publish_date(self) -> date:
         """
-        Calculate the SOPN publish date for a City of London local election.
+        Calculate the "SOPN publish date" for a City of London local election.
+
+        There are actually 2 releavant dates here:
+        The SOPN is _published_ **17 (working) days before polling day**
+        but then candidates can be withdrawn from it for an additional day
+        after that.
+        So the SOPN becomes _final_ **16 (working) days before polling day**
+
+        For the sake of simplicity, we are going to call 16 days before
+        polling day the "sopn_publish_date".
```

Comment:
For the sake of bashing City of London's square peg into the same round hole every other election goes through, this seems like the pragmatic choice.